File: tut-gradle-basic-java-project/src/test/java/com/github/galcyurio/MessageServiceTest.java
package com.github.galcyurio;

import org.junit.Before;
import org.junit.Test;

import static org.junit.Assert.*;

/**
 * @author galcyurio
 */
public class MessageServiceTest {

    private MessageService messageService;

    @Before
    public void setUp() throws Exception {
        messageService = new MessageService();
    }

    @Test
    public void getMessage_test() {
        String actual = messageService.getMessage();
        assertEquals("Hello World!", actual);
    }
}

File: tut-gradle-integration-test/src/test/kotlin/com/github/galcyurio/MessageServiceTest.kt
package com.github.galcyurio

import org.junit.Assert.assertEquals
import org.junit.Before
import org.junit.Test

/**
 * @author galcyurio
 */
class MessageServiceTest {

    lateinit var messageService: MessageService

    @Before
    fun setUp() {
        messageService = MessageService()
    }

    @Test
    fun `getMessage 단위 테스트`() {
        val actual = messageService.getMessage()
        assertEquals("Hello, world!", actual)
    }
}